sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -714,6 +732,7 @@ public partial class ServiceBusAdministrationClient
         public ServiceBusAdministrationClient(string fullyQualifiedNamespace, Azure.Core.TokenCredential credential) { }
         public ServiceBusAdministrationClient(string fullyQualifiedNamespace, Azure.Core.TokenCredential credential, Azure.Messaging.ServiceBus.Administration.ServiceBusAdministrationClientOptions options) { }
         public ServiceBusAdministrationClient(string connectionString, Azure.Messaging.ServiceBus.Administration.ServiceBusAdministrationClientOptions options) { }
+        public ServiceBusAdministrationClient(string fullyQualifiedNamespace, Azure.Messaging.ServiceBus.ServiceBusSharedAccessKeyCredential credential, Azure.Messaging.ServiceBus.Administration.ServiceBusAdministrationClientOptions options) { }
```

Comment:
Don't we need an overload without the options?

Reply:
Yes, we do.   Thanks!   This is an oversight on my part.